packages/oui-numeric/src/numeric.html

```diff
@@ -0,0 +1,31 @@
+<label ng-attr-for="{{:: $ctrl.id }}"
+       class="oui-label">
+  <span ng-bind=":: $ctrl.text"></span>
+  <span ng-if="$ctrl.unit"
+        ng-bind=":: '(' + $ctrl.unit + ')'"></span>
+</label>
+<div class="oui-input-group oui-input-group_numeric">
+  <button class="oui-button oui-button_icon-only oui-button_small-width"
+          type="button"
+          ng-disabled="$ctrl.disabled || $ctrl.model <= $ctrl.min"
+          ng-click="$ctrl.decrement()"
+          tabindex="-1"
```

Comment:
We really want to never tab on those buttons ?

Reply:
I think so, if you use keyboard and tab navigate to the component you will directly input the value. Then the next natural action will be to tab out to the next field. Buttons here are meant to be used with the mouse imho

Reply:
Yes, not necessary. 
On type number, you can do with the keyboard what the buttons do (up and down).
ex. https://www.w3schools.com/html/tryit.asp?filename=tryhtml_input_number

Reply:
Good point about the up and down key, didn't think about that :)